website_docs/configuration.md

```diff
@@ -0,0 +1,450 @@
+---
+title: "Configuration"
+weight: 20
+---
+
+Please be sure to review the following documentation:
+
+- [Data Collection concepts](../../concepts/data-collection) in order to
+  understand the repositories applicable to the OpenTelemetry Collector.
+- [Security
+  guidance](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security.md)
+
+## Basics
+
+The Collector consists of three components that access telemetry data:
+
+- <img width="32" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Receivers.svg"></img>
+[Receivers](#receivers)
+- <img width="32" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Processors.svg"></img>
+[Processors](#processors)
+- <img width="32" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Exporters.svg"></img>
+[Exporters](#exporters)
+
+These components once configured must be enabled via pipelines within the
+[service](#service) section.
+
+Secondarily, there are [extensions](#extensions), which provide capabilities
+that can be added to the Collector, but which do not require direct access to
+telemetry data and are not part of pipelines. They are also enabled within the
+[service](#service) section.
+
+An example configuration would look like:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  batch:
+
+exporters:
+  otlp:
+    endpoint: otelcol:55680
```

Comment:
It's probably about time to start using the new port everywhere, 4317 instead of 55680 :-)

Reply:
```suggestion
    endpoint: otelcol:4317
```

Reply:
Sorry, noob mistake. I think I got them all this time. @austinlparker feel free to merge in the suggestions if they look right.